src/main/scala/nl/knaw/dans/easy/stage/fileitem/UserCategory.scala

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (C) 2015-2016 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.stage.fileitem
+
+import nl.knaw.dans.common.lang.dataset.AccessCategory
+import nl.knaw.dans.common.lang.dataset.AccessCategory._
+
+object UserCategory extends Enumeration {
+  type UserCategory = Value
+  val
+  ANONYMOUS, // a user that is not logged in
+  KNOWN, // a logged in user
+  RESTRICTED_REQUEST, // a user that received permission to access the dataset
+  RESTRICTED_GROUP, // a user belonging to the same group as the dataset
+  NONE // none of the above
+  = Value
```

Comment:
i would like to rename (at some point in time) the AccessCategory to DatasetAccessCategory (or ~Type), because although it is in the dataset-package, this meaning is by no means obvious.
so i would suggest to rename this one to FileitemAccessCategory (or ~Type)